src/reducers/ChangePassword.reducer.ts

```diff
@@ -8,9 +10,9 @@ const defaultState = {
   loading: false,
 }
 
-export const changePassword = (state: any, action: any) => {
+export const changePassword = (state: any = defaultState, action: any = {}) => {
   switch (action.type) {
-    case Actions.ChangePassword.CHANGE_PASSWORD:
+    case onChangePassword().type:
```

Comment:
What is the benefit that brings to change it?

Reply:
Requires less code and imports if we grab the type from the action creator that we are already importing for the reducer tests.

It also means if we change the constant name that the action creator works on we don't have to fix the constant name in our reducer tests.

Reply:
and constants are annoying to type, so thats alwasy good ;D

Reply:
sound good, we will apply this pattern in all code?

Reply:
In the description Warren linked to Erics article recommends it: https://medium.com/javascript-scene/10-tips-for-better-redux-architecture-69250425af44

Reply:
Yup, if you are good with that?

Reply:
yes, sure